### DISCUSSION TITLE
Cart & Checkout Block: Fees: Add fee-ID as CSS-class

### DISCUSSION THREAD

**Discussion by larsenlarsson** [woocommerce/woocommerce#44998] [WooCommerce, Blocks, and Site Editing]
Hi folks 👋 

I hope you can help me again.

When the fees are rendered in the cart and checkout block I would like to add the fee->id as a class name.
Let's assume, we have this output:

![fees](https://github.com/woocommerce/woocommerce-blocks/assets/2648926/4da1bcdc-e917-4718-bcef-40c9afc016d4)

Then we have this rendered HTML:
![html](https://github.com/woocommerce/woocommerce-blocks/assets/2648926/2c09defb-3003-4f44-83a5-cc9a52061a2d)

I would like to add in each div
`<div class="wc-block-components-totals-item wc-block-components-totals-fees">`
the fee id, so that I would have these 3 DIVs:
```
<div class="wc-block-components-totals-item wc-block-components-totals-fees fee-1">
<div class="wc-block-components-totals-item wc-block-components-totals-fees fee-2>
<div class="wc-block-components-totals-item wc-block-components-totals-fees fee-3">
```

Is there a way to do this somehow?

Thanks in advance! 🙏

**Comment by senadir** [comment]
Seems like a reasonable request, the fee ID comes from 2 places:
- If it's passed to the Add Fee call (if you're directly using the Fees API class instead of the cart add_fee function).
- The title of the fee after being title sanatized. 

Just a note that using strings as classes is not recommended because they can be translated, they're language-dependent, what's `Fee` in english is `Frais` in french.

I'd susgest you:
- Open a PR if you have the know how (this is the fastest to get this merged).
- Open an issue for us to do it.

Otherwise, the class should be `wc-block-components-totals-fees__${ID}` instead of just `${ID}`, this would prevent collision.

**Reply by larsenlarsson** [reply]
Hi @senadir,

thank you for your time and your response.
As you suggested, I made my first pull request here:
https://github.com/woocommerce/woocommerce-blocks/pull/11054
I hope I created the pull request correctly...
